AddNewRecipes/Program.cs

```diff
@@ -415,7 +414,11 @@ public static void RunPatch(IPatcherState<ISkyrimMod, ISkyrimModGetter> state)
             modifiedList.Entries?.Add(mainpotionRecipeLVLIentry);
             state.PatchMod.LeveledItems.Set(mainpotionRecipeLVLI);
             Console.WriteLine("Adding recipes to defined containers");
-            IEnumerable<IContainerGetter> chests = from list in state.LoadOrder.PriorityOrder.OnlyEnabled().Container().WinningOverrides() where containerEditorIDs?.ToList().Contains(list.EditorID!) ?? true select list;
+            List<IContainerGetter> chests = new List<IContainerGetter>()
+            {
+                state.LinkCache.Resolve<IContainerGetter>(Skyrim.Container.TreasBanditChest),
+                state.LinkCache.Resolve<IContainerGetter>(Skyrim.Container.TreasDraugrChest),
+            };
```

Comment:
Few important concepts here.

Let's look at your original code, first.
```
HashSet<string> containerEditorIDs;

// .. later

containerEditorIDs?.ToList().Contains(list.EditorID!)
```

`HashSet<string>` is actually a good container to do Contains on.  It scales very well even when the number of items gets large.   But what are we doing here?  We're taking that HashSet, enumerating over all its values, making a new list with those values, and then doing Contains on that List.  That's a lot of extra work, and we're doing the Contains now on a List, which is a way worse container for Contains calls.  (can look up sets vs lists in general as to why)

------------

But even without the extra ToList call, this is not optimal.   Mutagen comes a concept called LinkCaches, which is a fancy version of `Dictionary<FormKey, MajorRecord>`.   Here we've swapped to using that instead, which is able to find your two intended records straight up via lookup, rather than via looping.   While Mutagen provides the LinkCache for us in this specific scenario, I would also recommend thinking about and using Dictionaries a lot more.  They're really nifty containers.

You'll also notice I'm making use of a nifty library that lets us write out EditorIds and get the FormKeys they're associated with, which is much more preferable to deal with when looking things up
<https://github.com/Mutagen-Modding/Mutagen.Bethesda.FormKeys>